### PR TITLE
Matching Channels with EPG

### PR DESCRIPTION
**Pull request by jkemp814**
Syncing channels with EPG list.

### CODE REVIEW

File: channels/us.m3u [hunk 1]
```diff
@@ -25,21 +25,21 @@ https://30a-tv.com/feeds/xodglobal/30atv.m3u8
...
 https://moiptvhls-i.akamaihd.net/hls/live/652312/secure/master.m3u8
 #EXTINF:-1 tvg-id="3ABNKids.us" tvg-name="3ABN Kids" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/eOvzf3d.png" group-title="Religious",3ABN Kids
 https://moiptvhls-i.akamaihd.net/hls/live/652318/secure/master.m3u8
-#EXTINF:-1 tvg-id="3ABNLatino.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
+#EXTINF:-1 tvg-id="K17JID4.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
 http://uni5rtmp.tulix.tv:1935/bettervida/bettervida/playlist.m3u8
-#EXTINF:-1 tvg-id="3ABNLatino.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
+#EXTINF:-1 tvg-id="K18JLD2.us" tvg-name="3ABN Latino" tvg-country="US" tvg-language="Spanish" tvg-logo="https://i.imgur.com/4ydVVIA.png" group-title="Religious",3ABN Latino
```
Comment (LaneSh4d0w) [inline channels/us.m3u:38]: Same programmation than the one above, but different call signs ?

**Comment by jkemp814** [comment]
Yes. There are about eight of those. Used different call signs to see which channel it might be base on time zones and content being played.

**Comment by LaneSh4d0w** [comment]
I see.

Even https://en.wikipedia.org/wiki/List_of_Three_Angels_Broadcasting_Network_affiliates that contains a lot of cities and their call signs is pretty confusing to read, when we're searching at the same time x)

File: channels/us.m3u (hunk 1 above)
Comment (jkemp814) [reply channels/us.m3u:38]: When streaming, some of them were showing different content. So, I'm trying to match them to the EPG. All have different call signs.

**Comment by Dum4G** [comment]
Am I approving it?

**Comment by jkemp814** [comment]
Yes I have to leave, so I will open a new one, later.

**Review by Dum4G — APPROVED** [review]

**Pull request merged**